2018/CreateAsyncFromSyncIterator.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+var AsyncFromSyncIterator = require('../helpers/AsyncFromSyncIterator');
+var Get = require('./Get');
+
+// https://www.ecma-international.org/ecma-262/9.0/#sec-createasyncfromsynciterator
+
+module.exports = function CreateAsyncFromSyncIterator(syncIteratorRecord) {
+	if (!AsyncFromSyncIterator) {
+		throw new SyntaxError('This environment does not support Promises.');
+	}
+
+	var asyncIterator = new AsyncFromSyncIterator(syncIteratorRecord);
```

Comment:
```suggestion
	var asyncIterator = AsyncFromSyncIterator(syncIteratorRecord);
```
i'm very confused why we'd want a constructor, or to encourage use of `new`.